packages/noports_core/lib/src/sshnp/util/sshrvd_channel/sshrvd_channel.dart

```diff
@@ -76,7 +76,7 @@ abstract class SshrvdChannel<T> with AsyncInitialization, AtClientBindings {
     // Connect to rendezvous point using background process.
     // sshnp (this program) can then exit without issue.
     Sshrv<T> sshrv = sshrvGenerator(
-      params.host,
+      host,
```

Comment:
We were using the wrong var here, host can be from sshrvd or falls back to params.host if there is no sshrvd